src/WebJobs.Script.WebHost/Diagnostics/FastLogger.cs

```diff
@@ -4,26 +4,33 @@
 using System;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host;
 using Microsoft.Azure.WebJobs.Host.Loggers;
 using Microsoft.Azure.WebJobs.Logging;
 using Microsoft.WindowsAzure.Storage;
-using Newtonsoft.Json;
 
 namespace Microsoft.Azure.WebJobs.Script.WebHost.Diagnostics
 {
     // Adapter for capturing SDK events and logging them to tables.
     internal class FastLogger : IAsyncCollector<FunctionInstanceLogEntry>
     {
         private readonly ILogWriter _writer;
+        private readonly TraceWriter _trace;
```

Comment:
You're right! I switched that around for the test at the last second and didn't fix it up.